_content/excerpt/mulamadhymakakarika_garfield-jay.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Mulamadhymakakarika"
+translator: garfield-jay
+external_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/187T2v8GMkuY7rT6Rl7a_oeXXSWKLn7u3/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
+  - nagarjuna
+  - madhyamaka
+  - mahayana
+  - emptiness
+year: 1995 
+oclc: 32012411 
+publisher: oxford
+address: "United Kingdom"
+booktitle: "The fundamental wisdom of the middle way: Nāgārjuna's Mūlamadhyamakakārikā"
+---
+A translation of a select seventy verses from Nāgārjuna's Mūlamadhyamakakārikā.
```

Comment:
This is good for a second paragraph, but this definitely deserves a pull quote. Perhaps,

```markdown
> By a misperception of emptiness  
A person of little intelligence is destroyed.
```